sys/include/net/gnrc/rpl.h

```diff
@@ -279,22 +284,25 @@ static inline bool GNRC_RPL_COUNTER_GREATER_THAN(uint8_t A, uint8_t B)
  * default lifetime will be multiplied by the lifetime unit to obtain the resulting lifetime
  * @{
  */
-#define GNRC_RPL_DEFAULT_LIFETIME (60)
-#define GNRC_RPL_LIFETIME_UNIT (2)
+#ifndef GNRC_RPL_DEFAULT_LIFETIME
+#define GNRC_RPL_DEFAULT_LIFETIME (5)
```

Comment:
First it corrects semantics of lifetime calculation, second it raises default lifetime to 5 min, which is as random as the 2 min before. However, 2 min seems to be rather low without knowing the actual network topology and its dynamics. Further this option is now exposed and hence configurable via Makefile, thus, if some developer/user has better knowledge about the network and/or requires a different setting it can be adapted.